daemon/src/model/cfd.rs

```diff
@@ -50,6 +51,8 @@ use time::OffsetDateTime;
 use uuid::adapter::Hyphenated;
 use uuid::Uuid;
 
+use super::FundingRate;
```

Comment:
Use absolute imports please.

Reply:
Still unaddressed :)

Reply:
sorry, rust-analyzer did this :) changed manually.